src/model/ModelStraightComponent.i

```diff
@@ -214,6 +215,7 @@ SWIG_MODELOBJECT(CoilHeatingDXMultiSpeed,1);
 SWIG_MODELOBJECT(CoilHeatingDXSingleSpeed,1);
 SWIG_MODELOBJECT(CoilHeatingDXVariableSpeed,1);
 SWIG_MODELOBJECT(CoilHeatingElectric,1);
+SWIG_MODELOBJECT(CoilHeatingElectricMultiStage,1);
```

Comment:
Definitely missing C# partial classes here.

Reply:
Then it was missing for `CoilHeatingGasMultiStage`...